styles/style.css

```
@@ -0,0 +1,83 @@
+@media only screen and (min-width: 375px) and (max-width: 768px) {
+  body {
+    margin: 0;
+    font-family: 'Inter', sans-serif;
+    overflow: hidden;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    padding-top: 280px;
+    padding-bottom: 280px;
+    background-image: url("/img/header-illsutration-mobile.svg");
```

- [ ] You may consider using the correct path for your background image.
```suggestion
    background-image: url("./img/header-illsutration-mobile.svg");
```